bin/checkstyle.sh

```diff
@@ -13,117 +13,117 @@
 
 # Return absolute path
 function realpath() {
-	local path="$1"
+        local path="$1"
```

Comment:
Do we really want to switch to spaces for shell scripts as well? The current .editorconfig supports the 8-character tab indentation type instead which is also what I'd prefer.

Reply:
Again, either way is fine - though file-level consistency is important.
I guess we could augment the checks to look for anything that ends in .sh or .csh or whose first line matches #!.*(sh,ksh,bash,csh,tcsh) - and substitute spaces for tabs (or vice versa).